### DISCUSSION TITLE
[General] <Running an op-challenger on the Sepolia testnet >

### DISCUSSION THREAD

**Discussion by AaronLee22** [ethereum-optimism/developers#521] [General]
### Discussion Topic

I'm currently running an op-challenger on the Sepolia testnet and trying to test some features. So far, I have successfully created a game. For the next step, I want to use the "move" subcommand. I have obtained the parent index through the list-claim command, but how can I get the claim itself?
![image](https://github.com/ethereum-optimism/developers/assets/114639805/4d046e3f-2769-4ff3-8a3d-fa272d115f80)

**Comment by AaronLee22** [comment]
It does work with the 'latest' but I would like to find out where I can find the PATENT_INDEX and CLAIM values.

**Reply by joohhnnn** [reply]
> It does work with the 'latest' but I would like to find out where I can find the PATENT_INDEX and CLAIM values.

have you tried this? 
```
./bin/op-challenger list-claims \
  --l1-eth-rpc https://rpc.ankr.com/eth\
  --game-address 0xb0b6aba1e035bbe14ca0affa77b6ed52e84ee7eb\
  --verbose
```

```
Idx Move    Parent Depth          Index Value                                                              Claimant                                     Bond (ETH) Time                Clock Used Resolution
  0 Attack             0     1073741823 0xcb4d30e4c8bb7ddf3a47afd861b786ef6d445580bd63d91d4b13494bc4b2c668 0x473300df21D047806A082244b417f96b32f13A33     0.080000 2024-07-11 09:46:59         0s ⏱️  2024-07-14 21:46:59
```

**Reply by AaronLee22** [reply]
Is the value  = CLAIM 
and 
Index = PATENT_INDEX

**Reply by joohhnnn** [reply]
> Is the value = CLAIM

Yes

> Index = PATENT_INDEX

No, it's [Trace Index ](https://github.com/ethereum-optimism/specs/blob/dfa8ea9568b0e35827be763fa8e6a2eeb9d90704/specs/fault-proof/stage-one/fault-dispute-game.md#position)

you can also find all the existing data from your game smart contract

**Reply by AaronLee22** [reply]
GOt it ! Thank you for your help!!!
